android/external/Makefile.conf

```diff
@@ -76,5 +80,5 @@ CC = $(ANDROID_NDK_HOME)/bin/$(COMPILER_PREFIX)$(ANDROID_API)-clang
 AS = $(CC)
 CXX = $(ANDROID_NDK_HOME)/bin/$(COMPILER_PREFIX)$(ANDROID_API)-clang++
 
-MYCFLAGS = -DANDROID -I$(LOCAL)/include $(TARGET_CFLAGS) -fPIE -fPIC -std=c99 $($ARCH_DEBUG_CFLAGS) # change 'release' to 'debug' for unoptimized debug builds
+MYCFLAGS = -DANDROID -I$(LOCAL)/include $(TARGET_CFLAGS) -fPIE -fPIC -std=c99 #$($ARCH_DEBUG_CFLAGS) # change 'release' to 'debug' for unoptimized debug builds
```

Comment:
Why is this out?

Reply:
just testing, it was running ok with that so it should not be an issue actually.

Reply:
It seems that I may be removing some flags that were needed from botan when I added the flags to find the jitter. This looks like will fix the performance issue.